### PR TITLE
Enable use of large Graviton 2 instances for AArch64 Linux builds on Travis

### PR DESCRIPTION
**Pull request by tanakadesilva**
Checklist
* [ x] Added a ``news`` entry

On Travis CI, use large Graviton 2 instance for AArch64 Linux builds, according to Amazon the newer instance should make builds up to two-times faster and allow us to build packages that currently timeout or results in OOM errors. The VM used for the build changes from 2 vCPUs to 4 vCPUs and RAM from 7.5 GB to ~16GB.

This is a one step in getting the polars feedstock to build on AArch64 Linux
https://github.com/conda-forge/polars-feedstock/pull/137

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Ideally, in my opinion, it would be using the Travis key.

The reservation was likely from when we included secrets in that key. The reservation is likely from that time.

Maybe for now you can use something like:

`travis_linux_aarch64_instance`

**Comment by tanakadesilva** [comment]
Thanks, updated the PR with those changes. Is there is anything else (tests, changes to docs) that I should do before the PR is marked as ready for review?

File: conda_smithy/configure_feedstock.py [hunk 1]
```diff
@@ -1707,6 +1707,10 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         "drone": {},
         "woodpecker": {},
         "travis": {},
+        "travis_linux_aarch64_instance": {
+            "name" : "arm64",
+            "size" : "medium",
+        },
```
Comment (isuruf) [inline conda_smithy/configure_feedstock.py:1713]: I don't think we need a separate key. We can do just like azure.

Comment (tanakadesilva) [reply conda_smithy/configure_feedstock.py]: Thanks, updated the PR based on your feedback.

**Comment by jaimergp** [comment]
Hi @tanakadesilva, do you know if Travis "charges" more credits for those instances? I can help you get this merged but we need to know whether this would incur in extra "costs" (faster usage of our open source credits) to check if this should be require a PR in `admin-requests` or not.

Thanks!

Tagging @conda-forge/core for visibility too.

**Comment by tanakadesilva** [comment]
Hi @jaimergp,

Thank you for pointing this out, I was not aware of the potential for additional credit usage associated with those instances on Travis.

The PR's goal was to build the rust components of polars for AAarch64 using Travis instances. I've noticed a recent [PR](https://github.com/conda-forge/polars-feedstock/pull/161) that address this by cross-compiling for AAarch64 on AMD64 using Azure and using a page file to avoid OOM errors. Given that there is no need for the changes in this PR I will close the PR. 

Thanks again for the feedback,

Tanaka

**Pull request closed without merging**